[0m[[0m[0mdebug[0m] [0m[0m> Exec(run, Some(8d38154a-e10c-49a7-aa43-f15967690644), Some(CommandSource(console0)))[0m
[0m[[0m[0mdebug[0m] [0m[0mEvaluating tasks: Compile / run[0m
[0m[[0m[0mdebug[0m] [0m[0mRunning task... Cancel: Signal, check cycles: false, forcegc: true[0m
[0m[[0m[0minfo[0m] [0m[0mcompiling 1 Scala source to /home/surzh/learning_scala/parse_zip/target/scala-2.13/classes ...[0m
[0m[[0m[33mwarn[0m] [0m[0m2 deprecations (since 2.13.0)[0m
[0m[[0m[33mwarn[0m] [0m[0m1 deprecation (since 2.13.3)[0m
[0m[[0m[33mwarn[0m] [0m[0m3 deprecations in total; re-run with -deprecation for details[0m
[0m[[0m[33mwarn[0m] [0m[0mthree warnings found[0m
[0m[[0m[31merror[0m] [0m[0mjava.io.FileNotFoundException: File does not exist: /user/surzh/gar_delta_xml.zip[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.INodeFile.valueOf(INodeFile.java:86)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.INodeFile.valueOf(INodeFile.java:76)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.FSDirStatAndListingOp.getBlockLocations(FSDirStatAndListingOp.java:156)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getBlockLocations(FSNamesystem.java:1990)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getBlockLocations(NameNodeRpcServer.java:768)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getBlockLocations(ClientNamenodeProtocolServerSideTranslatorPB.java:442)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:528)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1086)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1029)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:957)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.security.AccessController.doPrivileged(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at javax.security.auth.Subject.doAs(Subject.java:422)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1762)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2957)[0m
[0m[[0m[31merror[0m] [0m[0m[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.RemoteException.instantiateException(RemoteException.java:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.RemoteException.unwrapRemoteException(RemoteException.java:88)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.callGetBlockLocations(DFSClient.java:902)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.getLocatedBlocks(DFSClient.java:889)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.getLocatedBlocks(DFSClient.java:878)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.open(DFSClient.java:1046)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DistributedFileSystem$4.doCall(DistributedFileSystem.java:340)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DistributedFileSystem$4.doCall(DistributedFileSystem.java:336)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DistributedFileSystem.open(DistributedFileSystem.java:353)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.fs.FileSystem.open(FileSystem.java:976)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main$.delayedEndpoint$Main$1(Main.scala:26)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main$delayedInit$body.apply(Main.scala:15)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.Function0.apply$mcV$sp(Function0.scala:42)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.Function0.apply$mcV$sp$(Function0.scala:42)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.$anonfun$main$1(App.scala:98)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.$anonfun$main$1$adapted(App.scala:98)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:576)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:574)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.AbstractIterable.foreach(Iterable.scala:933)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.main(App.scala:98)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.main$(App.scala:96)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main$.main(Main.scala:15)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main.main(Main.scala)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.reflect.Method.invoke(Method.java:566)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.invokeMain(Run.scala:144)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.execute$1(Run.scala:94)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run$.executeSuccess(Run.scala:187)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.runWithLoader(Run.scala:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:1988)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1927)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.Try$.apply(Try.scala:213)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:367)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:829)[0m
[0m[[0m[31merror[0m] [0m[0mCaused by: org.apache.hadoop.ipc.RemoteException(java.io.FileNotFoundException): File does not exist: /user/surzh/gar_delta_xml.zip[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.INodeFile.valueOf(INodeFile.java:86)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.INodeFile.valueOf(INodeFile.java:76)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.FSDirStatAndListingOp.getBlockLocations(FSDirStatAndListingOp.java:156)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getBlockLocations(FSNamesystem.java:1990)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getBlockLocations(NameNodeRpcServer.java:768)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getBlockLocations(ClientNamenodeProtocolServerSideTranslatorPB.java:442)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:528)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1086)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1029)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:957)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.security.AccessController.doPrivileged(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at javax.security.auth.Subject.doAs(Subject.java:422)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1762)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2957)[0m
[0m[[0m[31merror[0m] [0m[0m[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Client.getRpcResponse(Client.java:1612)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Client.call(Client.java:1558)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Client.call(Client.java:1455)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:242)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.ProtobufRpcEngine2$Invoker.invoke(ProtobufRpcEngine2.java:129)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.sun.proxy.$Proxy34.getBlockLocations(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolTranslatorPB.getBlockLocations(ClientNamenodeProtocolTranslatorPB.java:333)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.reflect.Method.invoke(Method.java:566)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:422)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeMethod(RetryInvocationHandler.java:165)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invoke(RetryInvocationHandler.java:157)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.io.retry.RetryInvocationHandler$Call.invokeOnce(RetryInvocationHandler.java:95)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:359)[0m
[0m[[0m[31merror[0m] [0m[0m	at com.sun.proxy.$Proxy35.getBlockLocations(Unknown Source)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.callGetBlockLocations(DFSClient.java:900)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.getLocatedBlocks(DFSClient.java:889)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.getLocatedBlocks(DFSClient.java:878)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DFSClient.open(DFSClient.java:1046)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DistributedFileSystem$4.doCall(DistributedFileSystem.java:340)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DistributedFileSystem$4.doCall(DistributedFileSystem.java:336)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.fs.FileSystemLinkResolver.resolve(FileSystemLinkResolver.java:81)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.DistributedFileSystem.open(DistributedFileSystem.java:353)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.fs.FileSystem.open(FileSystem.java:976)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main$.delayedEndpoint$Main$1(Main.scala:26)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main$delayedInit$body.apply(Main.scala:15)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.Function0.apply$mcV$sp(Function0.scala:42)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.Function0.apply$mcV$sp$(Function0.scala:42)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:17)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.$anonfun$main$1(App.scala:98)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.$anonfun$main$1$adapted(App.scala:98)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.IterableOnceOps.foreach(IterableOnce.scala:576)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.IterableOnceOps.foreach$(IterableOnce.scala:574)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.collection.AbstractIterable.foreach(Iterable.scala:933)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.main(App.scala:98)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.App.main$(App.scala:96)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main$.main(Main.scala:15)[0m
[0m[[0m[31merror[0m] [0m[0m	at Main.main(Main.scala)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.reflect.Method.invoke(Method.java:566)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.invokeMain(Run.scala:144)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.execute$1(Run.scala:94)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.$anonfun$runWithLoader$5(Run.scala:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run$.executeSuccess(Run.scala:187)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Run.runWithLoader(Run.scala:121)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$bgRunTask$6(Defaults.scala:1988)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.Defaults$.$anonfun$termWrapper$2(Defaults.scala:1927)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:23)[0m
[0m[[0m[31merror[0m] [0m[0m	at scala.util.Try$.apply(Try.scala:213)[0m
[0m[[0m[31merror[0m] [0m[0m	at sbt.internal.BackgroundThreadPool$BackgroundRunnable.run(DefaultBackgroundJobService.scala:367)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.base/java.lang.Thread.run(Thread.java:829)[0m
[0m[[0m[31merror[0m] [0m[0m(Compile / [31mrun[0m) java.io.FileNotFoundException: File does not exist: /user/surzh/gar_delta_xml.zip[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.INodeFile.valueOf(INodeFile.java:86)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.INodeFile.valueOf(INodeFile.java:76)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.FSDirStatAndListingOp.getBlockLocations(FSDirStatAndListingOp.java:156)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.FSNamesystem.getBlockLocations(FSNamesystem.java:1990)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.server.namenode.NameNodeRpcServer.getBlockLocations(NameNodeRpcServer.java:768)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocolPB.ClientNamenodeProtocolServerSideTranslatorPB.getBlockLocations(ClientNamenodeProtocolServerSideTranslatorPB.java:442)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.hdfs.protocol.proto.ClientNamenodeProtocolProtos$ClientNamenodeProtocol$2.callBlockingMethod(ClientNamenodeProtocolProtos.java)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.ProtobufRpcEngine$Server$ProtoBufRpcInvoker.call(ProtobufRpcEngine.java:528)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.RPC$Server.call(RPC.java:1086)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:1029)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$RpcCall.run(Server.java:957)[0m
[0m[[0m[31merror[0m] [0m[0m	at java.security.AccessController.doPrivileged(Native Method)[0m
[0m[[0m[31merror[0m] [0m[0m	at javax.security.auth.Subject.doAs(Subject.java:422)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.security.UserGroupInformation.doAs(UserGroupInformation.java:1762)[0m
[0m[[0m[31merror[0m] [0m[0m	at org.apache.hadoop.ipc.Server$Handler.run(Server.java:2957)[0m
[0m[[0m[31merror[0m] [0m[0mTotal time: 17 s, completed Jun 11, 2024, 3:12:35 PM[0m
[0m[[0m[0mdebug[0m] [0m[0m> Exec(shell, None, None)[0m
[0m[[0m[0minfo[0m] [0m[0mshutting down sbt server[0m
